### PR TITLE
chore(javadoc): Change agent node to arm64

### PR DESCRIPTION
**Pull request by smerle33**
as per https://github.com/jenkins-infra/helpdesk/issues/3823#issuecomment-1961400721

migration to ARM64

Please do not merge

### CODE REVIEW

**Comment by smerle33** [comment]
green on ci: https://ci.jenkins.io/job/Infra/job/javadoc/job/PR-70/8/
and infra: https://ci.jenkins.io/job/Infra/job/javadoc/job/PR-70/8/

File: Jenkinsfile [hunk 1]
```diff
@@ -5,7 +5,8 @@ properties([
     pipelineTriggers([cron('H 5 * * 3')]),
 ])
 
-node('linux') {
+// 'linux-arm64' is now common between ci.jenkins.io and infra.ci.jenkins.io for VM "Docker Linux ARM64"
```
Comment (dduportal) [inline Jenkinsfile:8]:
```suggestion
```

(nit) I would remove the comment and update https://github.com/jenkins-infra/documentation/blob/main/ci.adoc instead

**Review by dduportal — APPROVED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (smerle33) [reply Jenkinsfile:8]: what about both ?
leaving the comment AND editing the doc ?

**Review by lemeurherve — APPROVED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (dduportal) [reply Jenkinsfile:8]: I don't see what is the point of the comment: the usual pipeline reader/writer is the developer of the project. THe comment here only drown them under information with no value.


Eventually we could link to the doc, but the current syntax does not provide any value

**Pull request merged**